.github/workflows/check-translations-and-deserialize.yml

```diff
@@ -141,7 +145,11 @@ jobs:
           team-reviewers: developer-enablement
 
       - name: Remove Completed Batches From Queue
-        if: steps.fetch-deserialize.outputs.batchesToDeserialize != 0
+        if: steps.fetch-deserialize.outputs.successfulTranslations != 0
```

Comment:
Same comment as the human-translated content: let's make sure we made a PR successfully before we clear the database.